FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -32,57 +32,57 @@
 #import <GoogleUtilities/GULAppEnvironmentUtil.h>
 #import <GoogleUtilities/GULSceneDelegateSwizzler.h>
 
-#import "FIRAdditionalUserInfo_Internal.h"
-#import "FIRAuthBackend.h"
-#import "FIRAuthCredential_Internal.h"
-#import "FIRAuthDataResult_Internal.h"
-#import "FIRAuthDispatcher.h"
-#import "FIRAuthErrorUtils.h"
-#import "FIRAuthExceptionUtils.h"
-#import "FIRAuthGlobalWorkQueue.h"
-#import "FIRAuthKeychainServices.h"
-#import "FIRAuthOperationType.h"
-#import "FIRAuthRequestConfiguration.h"
-#import "FIRAuthSettings.h"
-#import "FIRAuthStoredUserManager.h"
-#import "FIRAuthWebUtils.h"
-#import "FIRCreateAuthURIRequest.h"
-#import "FIRCreateAuthURIResponse.h"
-#import "FIREmailLinkSignInRequest.h"
-#import "FIREmailLinkSignInResponse.h"
-#import "FIREmailPasswordAuthCredential.h"
-#import "FIRGameCenterAuthCredential.h"
-#import "FIRGetOOBConfirmationCodeRequest.h"
-#import "FIRGetOOBConfirmationCodeResponse.h"
-#import "FIROAuthCredential_Internal.h"
-#import "FIRResetPasswordRequest.h"
-#import "FIRResetPasswordResponse.h"
-#import "FIRSendVerificationCodeRequest.h"
-#import "FIRSendVerificationCodeResponse.h"
-#import "FIRSetAccountInfoRequest.h"
-#import "FIRSetAccountInfoResponse.h"
-#import "FIRSignInWithGameCenterRequest.h"
-#import "FIRSignInWithGameCenterResponse.h"
-#import "FIRSignUpNewUserRequest.h"
-#import "FIRSignUpNewUserResponse.h"
-#import "FIRUser_Internal.h"
-#import "FIRVerifyAssertionRequest.h"
-#import "FIRVerifyAssertionResponse.h"
-#import "FIRVerifyCustomTokenRequest.h"
-#import "FIRVerifyCustomTokenResponse.h"
-#import "FIRVerifyPasswordRequest.h"
-#import "FIRVerifyPasswordResponse.h"
-#import "FIRVerifyPhoneNumberRequest.h"
-#import "FIRVerifyPhoneNumberResponse.h"
-#import "FirebaseAuth.h"
+#import <FirebaseAuth/FIRAuthSettings.h>
+#import <FirebaseAuth/FirebaseAuth.h>
```

Comment:
Isn't this the umbrella header? If so, doesn't it already include `FIRAuthSettings.h`?

Secondarily, shouldn't these be grouped with the other framework imports?